e2e/motoko/Main.mo

```diff
@@ -146,80 +147,128 @@ shared ({ caller = installer }) actor class Main() {
                 };
             };
 
-            let candidRpcCycles = 1_000_000_000_000;
-            let ethMainnetSource = #EthMainnet(?[#Alchemy, #Ankr, #Cloudflare, #BlockPi, #PublicNode]);
-
-            switch (await canister.eth_getBlockByNumber(ethMainnetSource, null, #Latest)) {
-                case (#Consistent(#Err(#ProviderError(#TooFewCycles _)))) {};
-                case result {
-                    addError("Received unexpected result: " # debug_show result);
-                };
-            };
-
-            Cycles.add(candidRpcCycles);
-            assertOk(
-                "eth_getLogs",
-                await canister.eth_getLogs(
-                    ethMainnetSource,
-                    null,
-                    {
-                        addresses = ["0xB9B002e70AdF0F544Cd0F6b80BF12d4925B0695F"];
-                        fromBlock = null;
-                        toBlock = null;
-                        topics = ?[
-                            ["0x4d69d0bd4287b7f66c548f90154dc81bc98f65a1b362775df5ae171a2ccd262b"],
-                            [
-                                "0x000000000000000000000000352413d00d2963dfc58bc2d6c57caca1e714d428",
-                                "0x000000000000000000000000b6bc16189ec3d33041c893b44511c594b1736b8a",
-                            ],
-                        ];
-                    },
+            let candidRpcCycles = 100_000_000_000;
+            let allServices : [(Text, EvmRpc.RpcServices)] = [
+                (
+                    "Ethereum",
+                    #EthMainnet(?[#Alchemy, #Ankr, #Cloudflare, #BlockPi, #PublicNode]),
                 ),
-            );
-            Cycles.add(candidRpcCycles);
-            assertOk(
-                "eth_getBlockByNumber",
-                await canister.eth_getBlockByNumber(ethMainnetSource, null, #Latest),
-            );
-            Cycles.add(candidRpcCycles);
-            assertOk(
-                "eth_getTransactionReceipt",
-                await canister.eth_getTransactionReceipt(ethMainnetSource, null, "0xdd5d4b18923d7aae953c7996d791118102e889bea37b48a651157a4890e4746f"),
-            );
-            Cycles.add(candidRpcCycles);
-            assertOk(
-                "eth_getTransactionCount",
-                await canister.eth_getTransactionCount(
-                    ethMainnetSource,
-                    null,
-                    {
-                        address = "0x1789F79e95324A47c5Fd6693071188e82E9a3558";
-                        block = #Latest;
+                (
+                    "Arbitrum",
+                    #Custom {
+                        chainId = 42161;
```

Comment:
How about defining constants for chain IDs?
If the constants are defined closer together, it is easier to verify the correctness of all used chain IDs.
If they are only used here, it's fine to leave it as it is, though.

Reply:
Only used here. I see your point though; I'll definitely refactor to constants if they end up being used in other places.